core/src/main/java/io/questdb/griffin/engine/functions/catalogue/PgAttributeFunctionFactory.java

```diff
@@ -43,7 +43,7 @@
 public class PgAttributeFunctionFactory implements FunctionFactory {
 
     private static final RecordMetadata METADATA;
-    private static final String SIGNATURE = "pg_catalog.pg_attribute()";
+    private static final String SIGNATURE = "pg_attribute()";
```

Comment:
Do we need to apply the same technique to other functions? We still have several functions that only have a prefix.

Reply:
Good finding. Tried to make them more consistent in d4791f7e3eff6a2f4eb01fc29cf57feedaabe25d